src/app/api/v1/fxa-rp-events/route.ts

```diff
@@ -191,7 +191,7 @@ export async function POST(request: NextRequest) {
           event
         ] as ProfileChangeEvent;
         logger.info("fxa_profile_update", {
-          subscriber: subscriber.id,
+          subscriber_id: subscriber.id,
```

Comment:
Nice cleanup, this is the kind of thing that can frustrate debugging sessions a lot.